fiwareskuld/expired_users.py

```diff
@@ -150,9 +151,9 @@ def _get_red_yellow(self, users):
                 # a week or less.
                 yellowList.append(user)
 
-        logger.info("Number of expired Trial Users found: %d",
+        logger.info("Number of expired Users found: %d",
```

Comment:
You can see that this methods has a list of users as input (they can be community, trials or whatever). Thus, the log should be changed.